src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -123,13 +123,7 @@ public RenameRewriter(RenameRewriterParameters parameters) : base(visitIntoStruc
                 _isProcessingComplexifiedSpans = false;
                 _skipRenameForComplexification = 0;
 
-                // TODO: These contexts are not changed for a document. ConflictResolver.Session should be refactored to cache them in a dictionary,
```

Comment:
All these data do not change during the renaming. So they are all cached in DocumentRenameInfo